CMakeLists.txt

```diff
@@ -80,19 +86,13 @@ option(FMT_TEST "Generate the test target." ${FMT_MASTER_PROJECT})
 option(FMT_FUZZ "Generate the fuzz target." OFF)
 option(FMT_CUDA_TEST "Generate the cuda-test target." OFF)
 option(FMT_OS "Include core requiring OS (Windows/Posix) " ON)
-option(FMT_MODULE "Build a module instead of a traditional library." OFF)
+cmake_dependent_option(FMT_MODULE "Build a module instead of a traditional library." OFF FMT_CAN_MODULE OFF)
```

Comment:
I don't think it should be a dependent option. Users should be able to override detection if needed.

Reply:
Alright, I've changed this to the simpler method, which is just excluding the MSVC, but still allowing the user to force overriding the `FMT_MODULE`.